demos/prefsFramework/index.html

```diff
@@ -93,14 +95,22 @@
         <div class="flc-overviewPanel fl-overviewPanel-container"></div>
         <!-- BEGIN markup for Preference Editor -->
         <div class="flc-prefsEditor-separatedPanel fl-prefsEditor-separatedPanel">
+            <!-- This div is for the sliding panel that shows and hides the Preference Editor controls in the mobile view-->
+            <div class="fl-panelBar fl-panelBar-above">
```

Comment:
What's the reason to have two identical `div`s for showing prefs editor in the mobile view and in the desktop view?